<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pairs</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {border-image: url(:/resources/Images/background.png) 0 0 0 0 stretch stretch;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="startPage">
       <property name="styleSheet">
        <string notr="true">QLabel { background-color : black; color : rgba(255,255,255,125); border-width: 1px; border-style: solid ; border-color: rgba(255,255,255,125)}
QPushButton {background-color: black; color: rgba(255,255,255,125);border-width: 1px; border-style: solid ; border-color: rgba(255,255,255,125)}
</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="menuGridLayout" columnstretch="0,0">
          <property name="leftMargin">
           <number>300</number>
          </property>
          <property name="topMargin">
           <number>200</number>
          </property>
          <property name="rightMargin">
           <number>300</number>
          </property>
          <property name="bottomMargin">
           <number>200</number>
          </property>
          <item row="1" column="1">
           <widget class="QSpinBox" name="playersSpinBox">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="startPushButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="quitPushButton">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="pairsLabel">
            <property name="text">
             <string>Pairs</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="playersLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Players</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="pairsSpinBox">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="titleLabel">
            <property name="font">
             <font>
              <family>Caladea</family>
              <pointsize>30</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Pairs </string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playerNamesPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color : rgba(255,255,255,125)}
QPushButton {background-color: black; color: rgba(255,255,255,125) ;border-width: 1px; border-style: solid ; border-color: rgba(255,255,255,125)}</string>
       </property>
       <layout class="QGridLayout" name="insertNamesLayout" rowstretch="1,0,0" columnstretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>250</number>
        </property>
        <property name="topMargin">
         <number>100</number>
        </property>
        <property name="rightMargin">
         <number>250</number>
        </property>
        <property name="bottomMargin">
         <number>100</number>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="1" column="0">
         <layout class="QGridLayout" name="playerNamesGridLayout" rowstretch="0" columnstretch="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>20</number>
          </property>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="buttonsGridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="gameOnPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Game on!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="enterNamesGridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gameboardPage">
       <property name="styleSheet">
        <string notr="true">QLabel{background-color : transparent ; color : rgba(255,255,255,125)}</string>
       </property>
       <layout class="QGridLayout" name="gameboardLayout" rowstretch="3,5" columnstretch="6,2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>10</number>
        </property>
        <item row="0" column="0" rowspan="2">
         <layout class="QGridLayout" name="gameBoardGridLayout" columnstretch="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="spacing">
           <number>10</number>
          </property>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="playerStatusGridLayout" rowstretch="0" columnstretch="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="topMargin">
           <number>300</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="scoresLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Caladea</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: transparent; 
color : rgba(255,255,255,125)</string>
            </property>
            <property name="text">
             <string>SCORES</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gameOverPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { color : rgba(255,255,255,125) }
QTextBrowser{color : rgba(255,255,255,125)}
QPushButton {background-color: black; color: rgba(255,255,255,125) ;border-width: 1px; border-style: solid ; border-color: rgba(255,255,255,125)}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>250</number>
        </property>
        <property name="topMargin">
         <number>200</number>
        </property>
        <property name="rightMargin">
         <number>250</number>
        </property>
        <property name="bottomMargin">
         <number>200</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gameOverGridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="gameOverMessageLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>2</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Caladea</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: black</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="gameOverLabel">
            <property name="font">
             <font>
              <family>Caladea</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Game Over!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="newGamePushButton">
            <property name="text">
             <string>New Game</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
